fetch_raw_data.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import json
+
+import pytz
+from core_data_modules.logging import Logger
+from core_data_modules.traced_data import Metadata, TracedData
+from core_data_modules.traced_data.io import TracedDataJsonIO
+from core_data_modules.util import IOUtils, TimeUtils, SHAUtils
+from id_infrastructure.firestore_uuid_table import FirestoreUuidTable
+from rapid_pro_tools.rapid_pro_client import RapidProClient
+from storage.google_cloud import google_cloud_utils
+from temba_client.v2 import Contact, Run
+
+from src.lib import PipelineConfiguration
+from src.lib.pipeline_configuration import RapidProSource
+
+Logger.set_project_name("Wusc-Keep-II")
```

Comment:
```suggestion
Logger.set_project_name("WUSC-KEEP-II")
```
(This is the project name on GitHub)

Reply:
The main difference being this does not support Shaqadoon and google cloud bucket as raw data sources.